sympy/stats/tests/test_rv.py

```diff
@@ -269,6 +269,15 @@ def test_dependent_finite():
     XX, YY = given(Tuple(X, Y), X + Y > 5)  # Create a dependency
     assert dependent(XX, YY)
 
+def test_dependent_pspace():
+    X, Y = Normal('X', 0, 1), Normal('Y', 0, 1)
+    assumps = Eq(Covariance(X, Y), S(1)/2)
+    rv = Dummy('rv', real=True)
+    assert density(X + Y, assumps).dummy_eq(
```

Comment:
`assumps` goes as `given`, right?

Reply:
I didn't get your question. But, currently, `assumps` go as `condition` to `density` and then without going to `given` it passes the call to `DependentPSpace`. Did you mean that?

Reply:
Well, as of now, I cannot do, 
```python
In [8]: X = Binomial('X', 5, S(1)/2)                                            

In [9]: Y = Binomial('Y', 6, S(1)/4)                                            

In [12]: P(X > 2, Eq(P(X > 1, Y > 1), S(1)/2))
ValueError: Overlapping Random Variables
```
In other words, queries like P(E|J) where E and J are some events dependent on each other isn't possible?

Reply:
Actually this will need some fixes after many changes, I will try to update this by tonight.

Reply:
Supporting these types of queries can be challenging.